### PR TITLE
fallback-message hiding moved to impress-common CSS file

### PR DESCRIPTION
**Pull request by simplePCBuilding**
I’m not home yet, but did the changes rq anyway. Will also close the issue. If anybody else has any suggestions, let me know

### CODE REVIEW

**Comment by simplePCBuilding** [comment]
I also had the suggestion to hide the fallback-message class from js. I could add that one if you’d like.

**Comment by simplePCBuilding** [comment]
Right. I kinda messed up there because I have not really worked with pull requests before

**Comment by fnogatz** [comment]
Then it's a good time to give it a second, fresh try? :)

**Comment by simplePCBuilding** [comment]
I'm removing all the commits from master and reopening the PR

File: css/impress-demo.css [hunk 1]
```diff
@@ -153,10 +153,6 @@ a:focus {
     margin-bottom: 10px;
 }
 
-.impress-supported .fallback-message {
-    display: none;
-}
-
```
Comment (henrikingo) [inline css/impress-demo.css:159]: you need to remove it in all other css files too. The ones under /examples/.

File: package-lock.json [hunk 2]
```diff
@@ -618,7 +618,6 @@
       "dependencies": {
         "anymatch": "~3.1.1",
         "braces": "~3.0.2",
-        "fsevents": "~2.1.2",
```
[the remark this answers is not in the document]
Comment (simplePCBuilding) [reply package-lock.json:621]: That modification was not intended

**Comment by henrikingo** [comment]
> I also had the suggestion to hide the fallback-message class from js. I could add that one if you’d like.

No, using CSS is correct. The fallback message is content. It may or may not exist, and user may use whatever HTML and CSS class around it.

Everything under impress-common.css is optional, while impress.js is of course not. (Although you could rebuild your own and leave out some plugins.) So as much as possible should be HTML+CSS because that leaves the end user the choice to do whatever they want.

**Comment by fnogatz** [comment]
> how do I reset my master branch?

Hey @simplePCBuilding, you could jump back to the latest commit of the original `impress/impress.js` repository, and create a new branch from that. Something like this:

```sh
cd your-local-repository
git checkout master
git checkout 64d66bf77feda50c6dcdf4d01bf5ef22a79e4df4 # this is the last commit your and the original impress/impress.js respositories have in common
git remote add upstream git@github.com:impress/impress.js.git # add the original impress/impress.js repository as upstream
git pull upstream master # pull latest changes from the original impress/impress.js repository
git checkout 840-move-fallback-message-css # create a new branch
# perform your changes and go on with git add + commit + push
```

**Comment by simplePCBuilding** [comment]
That didn't work.... It says it couldn't read from remote repo... yikes. Anyway. If we could merge the GettingStarted guide, I'd anyway have a clean master branch again. All this mess was caused due to me being new to PRs

**Comment by simplePCBuilding** [comment]
I will, once I have a clean master branch again, reopen this PR. I was not willing to delete my fork because I had already done a lot of work when it comes to the website, so I will just delete this branch sooner or later.

**Pull request closed without merging**